src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/MultiNodeTreePickerConverter.cs

```diff
@@ -100,7 +100,9 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
                 return multiNodeTreePicker;
             }
 
-            return base.ConvertFrom(context, culture, value);
+            // [ML] - return default list as its always nice to avoid null reference exceptions on lists =)
```

Comment:
I have the same concern as @JimBobSquarePants - having the base converter as fallback.

@micklaw - Yup, I'm sure we can workaround issues in the body of the `ConvertFrom`.